forge-game/src/main/java/forge/game/trigger/TriggerVote.java

```diff
@@ -66,29 +66,54 @@ public final boolean performTest(final Map<AbilityKey, Object> runParams) {
     @Override
     public final void setTriggeringObjects(final SpellAbility sa, Map<AbilityKey, Object> runParams) {
         @SuppressWarnings("unchecked")
-        FCollection<Player> voters = getVoters(
+        FCollection<Player> oppVotedDiff = getVoters(
             this.getHostCard().getController(),
             (ListMultimap<Object, Player>) runParams.get(AbilityKey.AllVotes),
-            true,
-            true
+            true, false
         );
-        sa.setTriggeringObject(AbilityKey.OtherVoters, voters);
+        sa.setTriggeringObject(AbilityKey.OpponentVotedDiff, oppVotedDiff);
+
+        FCollection<Player> oppVotedSame = getVoters(
+                this.getHostCard().getController(),
+                (ListMultimap<Object, Player>) runParams.get(AbilityKey.AllVotes),
+                true, true
+        );
+        sa.setTriggeringObject(AbilityKey.OpponentVotedSame, oppVotedSame);
     }
 
     @Override
     public String getImportantStackObjects(SpellAbility sa) {
         StringBuilder sb = new StringBuilder();
-        sb.append(Localizer.getInstance().getMessage("lblVoters")).append(": ").append(sa.getTriggeringObject(AbilityKey.OtherVoters));
+        if (hasParam("List")) {
```

Comment:
if the sa don't have List$ param defined, it's ok to return empty string? Cards like Capital Punishment, Coercive Portal?

Reply:
Hmm, all the `T:Mode$ Votes` do have the List param so I think this OK for now?

Reply:
are you sure its `$ Votes` 
![image](https://github.com/Card-Forge/forge/assets/9781539/30aea6fb-150f-4a06-bc6c-63eb387e4e5d)

not `$ Vote`
![image](https://github.com/Card-Forge/forge/assets/9781539/cab912e6-dcb8-4879-9780-42cd99a2045e)

Reply:
Ah sorry for the typo.

TriggerVote which corresponds to `T:Mode$ Vote` is the only thing that looks for `List` param.

VoteEffect which would be `SP$ Vote` or `DB$ Vote` doesn't access the TriggerVote code